src/components/transformation/create/InitializeDraftButton.tsx

```diff
@@ -101,7 +126,13 @@ function InitializeDraftButton({
                     setDraftId(draftId);
                     setPersistedDraftId(draftId);
 
-                    setOpenSQLEditor(true);
+                    if (setSQLEditorOpen) {
+                        setSQLEditorOpen(true);
+                    }
+
+                    // TODO (transform): Replace this with the navigate to create workflow hook and the production-ready URL
+                    //   when it is time to launch this feature.
+                    navigate(authenticatedRoutes.beta.new.fullPath);
```

Comment:
What are your thoughts on what this will look like when ready? Right now you get stuck when reloading on the `/new` path.

Reply:
It would function in a similar way to the edit workflows, using the drafted collection to prefill the form. The draft ID would be passed as a URL parameter as per usual.

Reply:
Okay.. so I am imagining something like : `collections/new?draftid=...`

Would we need to write a guard that pushes the consumer to the previous page if the draft isn't there? Or just land 'em on a page with a message of "this draft no longer exists"?